LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Beer-and-Code-Team
```

Comment:
colocar o nome correto da organização, Beer and Code (seguindo o nome cadastrado no perfil.